app/models/direct_file_data.rb

```diff
@@ -121,6 +123,14 @@ def phone_number=(value)
     write_df_xml_value(__method__, value)
   end
 
+  def cell_phone_number=(value)
+    write_df_xml_value(__method__, value)
+  end
+
+  def tax_payer_email=(value)
+    write_df_xml_value(__method__, value)
+  end
+
```

Comment:
added these so i could test that the list of attrs in the test also have setters (see direct_file_data_spec)